docs/reference-guides/interactivity-api/iapi-about.md

```diff
@@ -155,15 +155,13 @@ The API has been designed to be as performant as possible:
 
 Directives can be added, removed, or modified directly from the HTML. For example, users could use the [`render_block` filter](https://developer.wordpress.org/reference/hooks/render_block/) to modify the HTML and its behavior.
 
-In addition to using built-in directives, users can create custom directives to add any custom behaviors to their HTML.
-
 ### Atomic and composable
 
 Each directive controls a small part of the DOM, and you can combine multiple directives to create rich, interactive user experiences.
 
 ### Compatible with the existing block development tooling
 
-Using built-in directives does not require a build step and only requires a small runtime. A build step is necessary only when creating custom directives that return JSX. For such use cases, the API works out of the box with common block-building tools like [`wp-scripts`](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-scripts/).
+The API works out of the box with standard block-building tools like [`wp-scripts`](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-scripts/).
```

Comment:
@luisherranz @cbravobernal @sirreal What do you think about this change?
Are there any other ideas worth mentioning related to the "compatibility with the existing block development tooling"?

Reply:
It's worth mentioning that eight now you need to have `--experimental-modules` option.

Reply:
Thanks for the suggestion, Carlos.
I have added the requirement of `--experimental-modules` to this sentence and to the code requirements on the landing page of iAPI docs.